frontend/document/credocument.lua

```diff
@@ -458,8 +465,8 @@ function CreDocument:findText(pattern, origin, reverse, caseInsensitive)
 end
 
 function CreDocument:register(registry)
-    registry:addProvider("txt", "application/txt", self)
-    registry:addProvider("log", "application/txt", self)
+    registry:addProvider("txt", "text/plain", self)
+    registry:addProvider("log", "text/plain", self)
```

Comment:
IIRC, the mine type here is a koreader specific thing and is actually not used at all.